docs/managing-environments.md

```diff
@@ -76,6 +76,41 @@ special `system` Python version to retrieve the default behavior:
 poetry env use system
 ```
 
+## Activating the environment
+
+`poetry env activate` command prints the activate command to the console. This way you won't leave the current shell.
+You can then feed the output to `eval` to activate the environment. This way is the closest to manually activating the environment.
```

Comment:
I would restructure the text a bit and do not refer to `eval` but "the eval command of your shell" (or similar). Proposal:

```suggestion
The `poetry env activate` command prints the activate command of the virtual environment to the console.
You can run the output command manually or feed it to the eval command of your shell to activate the environment.
This way you won't leave the current shell.